### PR TITLE
updated config file

### PR DESCRIPTION
**Pull request by danihagosbl**

### CODE REVIEW

File: eshop-microservices/Services/Basket/Basket.API/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+# This stage is used when running from VS in fast mode (Default for Debug configuration)
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+USER app
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
+
+
```
Comment (danihagosbl) [inline eshop-microservices/Services/Basket/Basket.API/Dockerfile:10]: This blank space could be removed

File: eshop-microservices/Services/Catalog/Catalog.API/Data/ProductCatalogDbContext.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ public class ProductCatalogDbContext : DbContext
         public ProductCatalogDbContext(DbContextOptions<ProductCatalogDbContext> options)
             :base(options) { }
 
-        public DbSet<Product> Products { get; set; } = default!; 
+        public DbSet<Product> Productss { get; set; } = default!;
```
Comment (danihagosbl) [inline eshop-microservices/Services/Catalog/Catalog.API/Data/ProductCatalogDbContext.cs:11]: Remove this blank space

File: eshop-microservices/Services/Catalog/Catalog.API/Data/ProductCatalogDbContext.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@ public class ProductCatalogDbContext : DbContext
         public ProductCatalogDbContext(DbContextOptions<ProductCatalogDbContext> options)
             :base(options) { }
```
Comment (danihagosbl) [inline eshop-microservices/Services/Catalog/Catalog.API/Data/ProductCatalogDbContext.cs:9]: remove this space

**Review by danihagosbl — COMMENTED** [review]
Please have a look those request change

**Comment by danihagosbl** [comment]
Looks good for me

File: eshop-microservices/Services/Basket/Basket.API/Basket.API.csproj [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>7b5ca497-0eb2-4e4f-b6d9-2d74070407a3</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <DockerfileContext>..\..\..</DockerfileContext>
+  </PropertyGroup>
+
```
Comment (danielhagos11) [inline eshop-microservices/Services/Basket/Basket.API/Basket.API.csproj:11]: remove this blank space

File: eshop-microservices/Services/Basket/Basket.API/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+# This stage is used when running from VS in fast mode (Default for Debug configuration)
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+USER app
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
+
```
Comment (danielhagos11) [inline eshop-microservices/Services/Basket/Basket.API/Dockerfile:9]: remove this as well

File: eshop-microservices/Services/Basket/Basket.API/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+RUN dotnet restore "./Services/Basket/Basket.API/Basket.API.csproj"
+COPY . .
+WORKDIR "/src/Services/Basket/Basket.API"
+RUN dotnet build "./Basket.API.csproj" -c $BUILD_CONFIGURATION -o /app/build
+
+# This stage is used to publish the service project to be copied to the final stage
+FROM build AS publish
+ARG BUILD_CONFIGURATION=Release
+RUN dotnet publish "./Basket.API.csproj" -c $BUILD_CONFIGURATION -o /app/publish /p:UseAppHost=false
+
```
Comment (danielhagos11) [inline eshop-microservices/Services/Basket/Basket.API/Dockerfile:25]: remove this as well

**Review by danielhagos11 — APPROVED** [review]
Looks great to me Thanks Dan

**Pull request merged**